content/get-started/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Get Started with Webpack
+contributors:
+  - bebraw
+sort: 3
+---
+
+## Getting Started
+
+webpack is a tool to build javascript modules in your application. To start using `webpack` from its [cli](/api/cli) or [api](/api/node), follow the [Installation instructions](/get-started/install-webpack).
+webpack simplifies your workflow by quickly constructing a dependency graph of your application and bundling them in the right order. webpack can be configured to customise optimisations to your code, to split vendor/css/js code for production, have a hot reloaded dev server and many such cool features. Learn more about [why you should use webpack](/get-started/why-webpack).
+
+## Creating a bundle
+
+Create a demo directory to try out webpack. [Install webpack](/get-started/install-webpack).
+
+```bash
+mkdir webpack-demo && cd webpack-demo
+npm init -y
+npm install --save-dev webpack
+webpack --help //Shows a list of valid cli commands
+npm install --save lodash
+```
+
+Now create an `index.js` file.
+
+```javascript
+import _ from 'lodash';
+
+function component () {
+  var element = document.createElement('div');
+
+  element.innerHTML = _.map(['Hello','webpack'], function(item){
+    return item + ' ';
+  });
+
+  return element;
+};
+
+document.body.appendChild(component());
+```
+
+To run this piece of code, one usually has the below html
+
+```html
+<html>
+  <head>
+      <title>Webpack demo</title>
+      <script src='https://unpkg.com/lodash@4.16.6' type='text/javascript'/>
+      <script src='index.js' type='text/javascript'/> 
+  </head>
+  <body>
+  </body>
+</html>
+```
```

Comment:
You can also explain a bit more about dependencies management, such as:

> In this example, there are _implicit_ dependencies between the script tags.
>
> `index.js` depends on `lodash` being included in the page before it runs. It is implicit because `index.js` never said it needs to use `lodash`; it just assumes that a global variable `_` exists.
>
> There are problems with managing JavaScript projects this way: If a dependency is missing, or is included in the wrong order, the application won’t function. If a dependency is included but is not used, then there is an unnecessary code to download.

And in the example with webpack:

> Here, `index.js` explicitly requires `lodash` to be present, and binds it as `_` (no global scope pollution).
>
> By stating what dependencies a module needs, webpack can use this information to build a _dependency graph_, and generate an optimized bundle where scripts will be executed in the correct order, and unused dependencies will not be included in the bundle.

If you’d like to include a more extreme example, I have one from my old project, with [15 script tags](https://github.com/dtinth/IO-MIDI-INSTRUMENT/blob/ec819d4b60f3a1efb627e0da642981cdbf6fb14d/instruments/keyboard/index.html#L18-L33) that implicitly depended on each other 😛.